docs/developer-guide/testing.md

```diff
@@ -24,8 +24,42 @@ gobopt='--set ana.base.privatization write+lock' ./scripts/update_suite.rb
 ```
 
 ### Writing
-* Add parameters to a regression test in the first line: `// PARAM: --set warn.debug true`
-* Annotate lines inside the regression test with comments: `arr[9] = 10; // WARN`
+Regression tests use single-line comments (with `//`) as annotations.
+
+#### First line
+A comment on the first line can contain the following:
+
+| Annotation | Comment |
+| ---------- | ------- |
+| `PARAM: ` <br> (NB! space) | The following command line parameters are added to Goblint for this test. |
+| `SKIP` | The test is skipped (except when run with `./scripts/update_suite.rb group`). |
```

Comment:
```suggestion
| `SKIP` | The test is skipped (except when run with `./scripts/update_suite.rb group`).  This is used e.g. in the tests for Apron, as not all CI jobs have the Apron library installed. |
```

Reply:
I added this note above where we describe the test running scripts. Too long text stretches out these tables oddly.